pandas/tests/series/test_datetime_values.py

```diff
@@ -509,3 +509,8 @@ def test_dt_timetz_accessor(self, tz_naive_fixture):
                            time(22, 14, tzinfo=tz)])
         result = s.dt.timetz
         tm.assert_series_equal(result, expected)
+
+    def test_minmax_nat(self):
```

Comment:
can u add test for timedelta dtype and DataFrame (#10390)

Reply:
Added more tests, but this PR does not fix #10390